Userland/Services/WebContent/PageHost.cpp

```diff
@@ -151,6 +151,7 @@ void PageHost::page_did_start_loading(const URL& url)
 
 void PageHost::page_did_finish_loading(const URL& url)
 {
+    m_client.initialize_js_console({});
```

Comment:
Maybe add a `Fixme` there again, similar to the one you removed? There's still something to fix after all :^)

Reply:
:+1: